app/helpers/admin/products_helper.rb

```diff
@@ -11,19 +11,20 @@ def product_image_form_path(product)
     end
 
     def prepare_new_variant(product)
```

Comment:
We could remove the helper has it's not doing much now, but that's fine as it is.